packages/sandbox/ui/src/store-meta/store.ts

```diff
@@ -20,6 +20,7 @@ if (process.env.NODE_ENV === 'development') {
     )
   )
 } else {
+  // @ts-ignore
```

Comment:
After upgrading TypeScript I started getting:
```
error TS2589: Type instantiation is excessively deep and possibly infinite.
```
I tracked it down to a loop in types: https://github.com/nextools/metarepo/blob/master/packages/typeon/src/index.ts#L3
`TJsonMap` uses `TJsonValue` that can be `TJsonMap`. So, in this case, TypeScript is not wrong, it's a very deep and possibly infinite type, but it's what we want, since we allow to nest meta files without limitation.
Because nothing, besides typescript version, changed that could affect this setup, I decided to ignore it.